internal/cli/clients/client.go

```diff
@@ -417,12 +417,12 @@ func (c *EpinioClient) DeleteService(name string, unbind bool) error {
 			// and the response contains an array of their
 			// names.
 
-			var apiError map[string][]api.APIError
+			var apiError api.ErrorResponse
 			if err := json.Unmarshal(bodyBytes, &apiError); err != nil {
 				return err
 			}
 
-			bound := strings.Split(apiError["errors"][0].Details, ",")
+			bound := strings.Split(apiError.Errors[0].Details, ",")
```

Comment:
Isn't there a case when `apiError.Errors` has no elements? Wouldn't this fail then? (the old code would have this issue too, I just wonder how we can be sure that `[0]` here exists)

Reply:
The underlying structures are for one or more errors. (APIError, MultiError).
This can only happen if it is possible to generate a MultiError with zero elements.
@manno Is that possible ?

Reply:
Sure, we could construct an ErrorResponse without an error in the API server. You would have to use MultiError in an action. 

While you could create such a struct, MultiError itself needs at least one error element.
Here is the only user of MultiError: https://github.com/epinio/epinio/blob/main/internal/api/v1/servicebindings.go#L85-L117

I don't think that code can return an empty one, see the check at the end.